code/main/edge/buildEdgeRequestURL.brs

```diff
@@ -13,12 +13,22 @@
 
 ' ***************************** MODULE: buildEdgeRequestURL *******************************
 
-function _adb_buildEdgeRequestURL(configId as string, requestId as string, edgeDomain = invalid as dynamic) as string
+function _adb_buildEdgeRequestURL(configId as string, requestId as string, path as string, edgeDomain = invalid as dynamic) as string
     scheme = "https://"
     host = "edge.adobedc.net"
-    path = "/ee/v1/interact"
+    overridablePath = "/ee/v1/interact"
```

Comment:
Let's extract ee and v1 into constants like we do in the Mobile case

```
REQUEST_URL_VERSION = "/v1"
REQUEST_URL_PROD_PATH = "/ee"
```

Reply:
Refer: 
https://github.com/adobe/aepsdk-edge-android/blob/main/code/edge/src/main/java/com/adobe/marketing/mobile/EdgeEndpoint.java#L70-L116

Reply:
What's the reason we have to use two constants here? If it's not required for now, In my opinion, we should consider doing it when necessary in the future.

Reply:
so ee can be used even with paths. We can just get the `/va/v1/MediaEvent` instead of `ee/va/1/MediaEvent`

Reply:
I think it's a bit overkill here. I can't see a benefit. If `ee` can be replaced, I think we can make the change.

Reply:
The reason I added this was since ee is added to normal edge and media as well, so we would not replace ee just to be replaced with ee prefixed path, but it's okay to not add this change for now.

Reply:
Can we make it the responsibility of the caller to pass the complete path and not have any additional logic here?

Reply:
Yeah, I think it makes sense always to let the caller pass the path since we may support embedding the location hit in the URL.